pages/wi/2021/jobs.js

```diff
@@ -142,17 +151,17 @@ const jobs = () => {
   return (
     <>
       <NextSeo
-        title="WI 2020 Jobs - THAT Conference"
+        title="WI 2021 Jobs - THAT Conference"
         description="Job openings for all WI 2020 THAT partners"
       />
 
       <ContentSection>
         <Main>
           <SideDetail>
-            <h1>THAT WI 2020 - Partner Jobs</h1>
+            <h1>THAT WI 2021 - Partner Jobs</h1>
```

Comment:
Possible to be dynamic?

Reply:
@brettski they are in pages/wi/2021/jobs....so dynamic doesn't really make sense.

Reply:
Sure it does, less to update when creating 2022, I would think the end goal is that we are not copy directories around to make a new event...

Reply:
well changing that date to be dynamic when the directory is not doesn't address the larger issue if the goal is to not have to manually create/copy a 2021 directory.